text-editor/text-editor-002-en.md

```diff
@@ -170,17 +176,19 @@ document.execCommand('bold')
 and that's it ! Your selection is wrapped inside a `b` tag, the produced
 markup is very clean (remember the previous step where we had to sanitize the
 markup manually).
+
 This is very neat, it enables rich text edition directly in the browser (and
-by the browser :p).
+by the browser :p). And look at that compatibility table, green everywhere ! The
+icing on the cake: conventional shortucts work as well (ctrl+B for bold, etc).
```

Comment:
Actually, I don't think such sortcuts are part of the `document.execCommand` stuff, but rather a default binding of the browser for keyboard shortcuts on  `contenteditable` elements. (they may however use the same loginc under the hood than the one for `document.execCommand`).

Reply:
You're absolutely right.